### PR TITLE
Add previous PR

### PR DESCRIPTION
**Pull request by ddomingof**
Add new families:

- 5-HT
- AGTR
- CYP

### CODE REVIEW

**Comment by ddomingof** [comment]
@bgyori @steppi it should be fine now

File: equivalences.csv [hunk 1]
```diff
@@ -11,8 +11,8 @@ BEL,Adaptor Protein II Complex,Adaptor_protein_II
 BEL,Adaptor Protein III Complex,Adaptor_protein_III
 BEL,ADCY Family,ADCY
 BEL,ADH Family,ADH
-BEL,ADRA Family,ADRA
 BEL,ADRA2 Family,ADRA2
+BEL,ADRA Family,ADRA
```
Comment (bgyori) [inline equivalences.csv:15]: Many of the old entries were reordered in this file, let's try to revert the changes for existing entries and just add the new ones.

File: equivalences.csv [hunk 2]
```diff
@@ -432,17 +432,45 @@ GO,GO:0048180,Activin
 GO,GO:0070652,Augmin
 GO,GO:0070765,Gamma_secretase
 GO,GO:1990779,GPIb_IX_V
+HGNC_GROUP,1000,CYP1
```
Comment (bgyori) [inline equivalences.csv:435]: The sorting here was by number i.e., `2` was before `1000`, this is changed by this PR, we could go back to the old ordering.

File: entities.csv [hunk 3]
```diff
@@ -1,3 +1,5 @@
+5-HT
```
Comment (bgyori) [inline entities.csv:1]: As a convention, we've used underscores for FamPlex entries so following that logic, this would be `5_HT`.

File: relations.csv [hunk 4]
```diff
@@ -4067,3 +4068,104 @@ HGNC,MIRLET7A3,isa,FPLX,MIRLET7A
...
+FPLX,5_hydroxytryptamine_receptors_G_protein_coupled,isa,FPLX,5-HT
+HGNC,HTR1A,isa,FPLX,HTR1
+HGNC,HTR1B,isa,FPLX,HTR1
+HGNC,HTR1D,isa,FPLX,HTR1
+HGNC,HTR1E,isa,FPLX,HTR1
+HGNC,HTR2A,isa,FPLX,HTR2
+HGNC,HTR2B,isa,FPLX,HTR2
+HGNC,HTR2C,isa,FPLX,HTR2
+HGNC,HTR5A,isa,FPLX,HTR5
+HGNC,HTR5BP,isa,FPLX,HTR5
```
Comment (bgyori) [inline relations.csv:4086]: `HTR5BP` is a pseudogene, we could remove it. That would leave `HTR5A` as the only member of `HTR5` meaning that we can also remove `HTR5` and link `HTR5A` directly to `HTR5`'s parent.
